index.html

```diff
@@ -339,6 +339,11 @@ <h5>ActionScript 3 API <code>5%</code></h5>
 					</p>
 				</div>
 			</div>
+			<div class="row">
+				<div class="col-base col-lg d-flex justify-content-center">
+					<canvas id="weeklyCommits"></canvas>

```

Comment:
I think the naming convention is [`kebab-case`](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) - `weekly-commits`